src/service/performance-impl.js

```diff
@@ -59,6 +59,20 @@ function incOrDef(obj, name) {
   }
 }
 
+/**
+ * Get the visibility state of the provided document.
+ *
+ * @param {Document} document
+ */
+function getVisibilityState(document) {
```

Comment:
The implementation of this function suggests you're getting lifecycle states rather than visibility states. Since I don't see that you're using the `active` or `passive` states anywhere else in the code, you could probably remove this and just use `document.visibilityState`.

Reply:
Done.